zoo-css-challenge/style.css

```
@@ -160,12 +191,23 @@ footer {
   background-size: cover;
   background-position: center;
   padding: 100px 5%;
+  border-radius: 10px;
 }
 
 .education-container {
   background-color: rgba(255, 255, 255, 0.75);
   padding: 50px;
   max-width: 400px;
+  border-radius: 10px;
+}
+
+.education-container button {
+  font-size: 1rem;
+}
+
+.education-container button:hover, button:focus {
```

```suggestion
.education-container button:hover, .education-container button:focus {
```
Similar to my other comment, we want `button:focus` to apply only within `.education-container`